src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertListItem.tsx

```diff
@@ -26,29 +15,44 @@ interface SavedSearchAlertListItemProps {
 
 export const SavedSearchAlertListItem: React.FC<SavedSearchAlertListItemProps> = ({
   item,
-  variant,
+  variant = "inaclive",
   onEditAlertClick,
 }) => {
   const viewAllHref = item.href
-  const [isExpanded, setIsExpanded] = useState(false)
-  const Icon = isExpanded ? ChevronUpIcon : ChevronDownIcon
+  const matchingArtworksCount = item.artworksConnection?.counts?.total
 
-  const toggleExpandFilters = () => setIsExpanded(isExpanded => !isExpanded)
-
-  const toggleExpandFiltersText = isExpanded
-    ? "Close all filters"
-    : "Show all filters"
+  const TextWrapper = ({ children, textVariant }) => (
+    <>
+      <Media greaterThanOrEqual="md">
+        <Text
+          variant={textVariant}
+          color={variant === "active" ? "black100" : "black60"}
+          style={{ overflowWrap: "break-word" }}
+        >
+          {children}
+        </Text>
+      </Media>
+      <Media lessThan="md">
+        <Text
+          variant={textVariant}
+          color="black100"
+          style={{ overflowWrap: "break-word" }}
+        >
+          {children}
+        </Text>
+      </Media>
+    </>
```

Comment:
Would this work as well? Like this, it could be just inline, and we wouldn't need an extra text wrapper component for it.

```suggestion
        <Text
          variant={textVariant}
          color={["black100", variant === "active" ? "black100" : "black60"]}
          style={{ overflowWrap: "break-word" }}
        >
          {children}
        </Text>
```

Reply:
I tried, the color can't take an array unfortunately, ts complains

Reply:
there is a textColor property that has a ResponsiveValue interface, will it work with it? https://github.com/artsy/palette/blob/main/packages/palette/src/elements/Text/Text.tsx#L34

Reply:
Do we actually need to check for device size here to change color? In the mWeb list view all items will be always inactive (with black100 color), right? I think it's safe to keep only:

```
        <Text
          variant={textVariant}
          color={variant === "active" ? "black100" : "black60"}
          style={{ overflowWrap: "break-word" }}
        >
          {children}
        </Text>
```

without Media

@dariakoko what do you think?

Reply:
It would look so much better, but we need all item inactive and 'black100' for mWeb, when for Web we need inactive 'black60' and active 'black100'

Reply:
Wait' will try out textColor

Reply:
It woooorks! @nickskalkin 🥳 🥳